src/sgame/botlib/bot_types.h

```diff
@@ -60,6 +62,9 @@ struct botNavCmd_t
 	float    dir[ 3 ]; // normalized direction of the target
 	int      directPathToGoal;
 	int      havePath;
+	glm::vec3 glm_dir ( void ) const { return glm::vec3( dir[0] , dir[1] , dir[2] ); };
```

Comment:
Do not write `void` in function prototypes.

Reply:
Fixed

Reply:
There's still some `( void )` left in there

Reply:
Right, removed the last one. Trusting github to show the whole PR was a bad idea :S